bedrock/firefox/views.py

```diff
@@ -541,6 +542,17 @@ def get_template_names(self):
         return [template]
 
 
+def fxos(request):
+    locale = l10n_utils.get_locale(request)
+    old_home = 'firefox/os/index.html'
+    new_home = 'firefox/os/index-new.html'
+
+    if lang_file_is_active(new_home, locale):
```

Comment:
`lang_file_is_active()` requires you pass it a lang file name. Passing it a template name won't work. If you really want to see if a template will be active for a locale use `lib.l10n_utils.gettext.template_is_active`.